### ISSUE TITLE
Delete unused KPIMON xApps from OAIC project

### ISSUE THREAD

**Issue by jgaeddert** [openaicellular/oaic#115]
There are several existing KPIMON xApps (repositories not included as submodules) under OAIC, but I think only one is being used. Let's remove the ones we aren't using any more.

**Comment by joshuamoorexyz** [comment]
It looks like i would have to be an admin for the organization to delete the unused repo.

**Comment by jgaeddert** [comment]
B'oh. Sorry. Can you confirm that these repos can be removed?

  * https://github.com/openaicellular/upgraded-kpimon-xApp
  * https://github.com/openaicellular/scp-ric-app-kpimon

Side question, it looks like we have a lot of "test xapp" repos as well. Any clue which ones we need to keep?

  - [ ] https://github.com/openaicellular/test_xapp
  - [ ] https://github.com/openaicellular/oaict-test-xapp
  - [ ] https://github.com/openaicellular/oaic-samplexapp-test1

**Comment by joshuamoorexyz** [comment]
Yes those are safe to remove. I will also unlink the workshop instructions as to not build them into our documentation as well. @mrkouchaki can you comment about the test xapps?

**Comment by mrkouchaki** [comment]
This oaict-test-xapp: 'https://github.com/openaicellular/oaict-test-xapp' should be relocate to master branch: 'https://github.com/openaicellular/oaic' or inside the oaic-t repo in master branch: 'https://github.com/openaicellular/oaic-t/tree/16709d018ccd961e2038dfa2e61d7baf5091a47e' , it is necessary for oaic-t deployment.

Also for KPIMON we need this 'https://github.com/openaicellular/upgraded-kpimon-xApp' version in master branch both for oaic-t and oaic deployments.
